thor/tests/test_observation_source.py

```diff
@@ -0,0 +1,125 @@
+import astropy.time
+import numpy as np
+import pyarrow as pa
+import pytest
+import quivr as qv
+from adam_core import coordinates, observers, propagator
+from adam_core.observations import detections, exposures
+
+from .. import observation_source, orbit
+
+
+@pytest.fixture
+def fixed_test_orbit():
+    # An orbit at 1AU going around at (about) 1 degree per day
+    coords = coordinates.CartesianCoordinates.from_kwargs(
+        x=[1],
+        y=[0],
+        z=[0],
+        vx=[0],
+        vy=[6.14],
```

Comment:
6.14 au per day is ~3% the speed of light. This test orbit is flying!

You could give it a y-velocity of `np.sqrt(c.MU/r)` if you wanted to make it a circular orbit instead of an unbound one.

Reply:
Hahahaha, yeah, that's what what I actually wanted! Thanks.

Reply:
(I'm doing 2𝜋/365.24 instead, because that's how my brain works, and they're the same up to 5 digits of precision :))